tox.ini

```diff
@@ -1,15 +1,16 @@
-
 [tox]
 minversion = 1.8
 envlist =
-    py{27}-ansible{24,25,26}-functional
-    py{27,36,37}-ansible{24,25,26}-unit
-    py{36,37}-ansible{24,25,26}-functional
+    py{27,36,37}-ansible{24,25,26}-functional
+    py{27,36,27}-ansible{24,25,26}-unit
     py{27,36,37}-lint
     format-check
     doc
 skipdist = True
 skip_missing_interpreters=True
+# sitepackages improve speed and footprint at small risk of missing to spot
+# undeclared dependencies which are already installed on the system.
+sitepackages=True
```

Comment:
I don't think we should do this. Tests are already unstable. No more cumulative risks, please.

Reply:
sure, better to do this isolated if we do it.